.github/workflows/cla.yml

```diff
@@ -0,0 +1,35 @@
+name: "CLA Assistant"
+
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened,closed,synchronize]
+
+permissions:
+  actions: write
+  contents: read
+  pull-requests: write
+  statuses: write
+
+jobs:
+  CLAAssistant:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "CLA Assistant"
+        if: (github.event.comment.body == 'recheck' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'
```

Comment:
I'd prefer this to be a check like `contains(github.event.comment.body, '@PasteurBot')` to avoid enforcing exact matches. wdyt?

Reply:
This is the recommended setup for the action, but we can tweak.

The `if` should be `true` when:
- a comment in the PR is exactly `recheck`, to re-check the CLA signatures;
- a comment in the PR is a user signing the CLA with exactly this: `I have read the CLA Document and I hereby sign the CLA`;
- a PR is opened, closed, or updated.

So, I'd keep it as is.